modules/sociallogin/sociallogin.admin.view.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+class SocialloginAdminView extends Sociallogin
+{
+	function init()
+	{
+		Context::set('config', self::getConfig());
+
+		if ($this->module_config->delete_auto_log_record)
+		{
+			$args = new stdClass;
+			$args->regdate_less = date('YmdHis', strtotime(sprintf('-%d day', $this->module_config->delete_auto_log_record)));
+			executeQuery('sociallogin.deleteLogRecordLess', $args);
+		}
+
+		$this->setTemplatePath($this->module_path . 'tpl');
+
+		Context::addJsFile($this->module_path . 'tpl/js/sociallogin_admin.js');
+	}
+
+	function dispSocialloginAdminSettingApi()
+	{
+		Context::set('can_be_use_twitter', version_compare(PHP_VERSION, '7.3.0', '>='));
```

Comment:
```suggestion
		Context::set('sns_twitter_use_cond', version_compare(PHP_VERSION, '7.3.0', '>='));
```
+버전체크는 init쪽으로 옮겨도 될것 같습니다.